wpilibj/src/athena/java/edu/wpi/first/wpilibj/DigitalGlitchFilter.java

```diff
@@ -52,10 +52,14 @@ public void free() {
   }
 
   private static void setFilter(DigitalSource input, int channelIndex) {
-    if (input != null) { // Counter might have just one input
-      DigitalGlitchFilterJNI.setFilterSelect(input.m_port, channelIndex);
+    if (input != null) { // Counter might have just oe input
+      // analog triggers are not supported for DigitalGlitchFilters
+      if (input.getAnalogTriggerForRouting()) {
+        return;
```

Comment:
Should this generate some kind of error/warning instead of just silently ignoring?